server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaBuilder.java

```diff
@@ -520,6 +523,22 @@ public CentralDogmaBuilder writeQuotaPerRepository(int writeQuota, int timeWindo
         return this;
     }
 
+    /**
+     * Sets the configuration for retaining commits in a {@link Repository}.
+     * The {@link Repository} retains at least the number of {@code minRetentionCommits} when more than
+     * {@code minRetentionCommits} are made.
+     * The {@link Repository} also retains commits at least {@code minRetentionDays}. If both conditions are
+     * not satisfied, the commits are not removed.
+     * For example, when {@code minRetentionCommits} is set to 2000 and {@code minRetentionDays} is set to 14,
+     * the commits that are created more than 2000 are not removed until 14 days have passed. Set 0 to retain
+     * all commits.
```

Comment:
I'm a bit confused with the policy when both `minRetentionCommits` and `minRetentionDays` are set.

> the commits that are created more than 2000 are not removed until 14 days have passed.

IIUC, if commits seem not to be removed in 14 days even if the number of commits exceeds 2000.
2000 `minRetentionCommits` with 14 `minRetentionDays` sounds like Central Dogma allows 2000 commits for 14 days.
If commits are created more than 2000 in 14 days, some old commits need to be removed. Although the old commits were less than 14 days old.